README.md

```diff
@@ -15,9 +15,10 @@ If you need help with anything CNCF + Summer of code related, you can file an is
 Communication
 -------------
 
-It's best if you use a public communication channel whenever possible; however, if you need to communicate in private, please feel free to send the admins a note via soc@cncf.io (please use the public channels for any project-related discussion).
+Please reach out to us on the #gsoc channel on the [CNCF slack](https://slack.cncf.io/).
```

Comment:
May point to https://cloud-native.slack.com/messages/gsoc

Reply:
Thought of doing that...but that page doesn't tell how to sign up for the slack and I've seen lots of questions where students ask how to sign up. :/

Reply:
I see, +1 :)